### PR TITLE
Add comment and example to option 'associated_property'.

### PR DESCRIPTION
**Pull request by tim96**

### CODE REVIEW

**Comment by greg0ire** [comment]
Please rebase on the latest master

File: Resources/doc/reference/action_list.rst [hunk 1]
```diff
@@ -72,12 +72,16 @@ Here is an example from Sonata E-Commerce Product Admin:
             ))
 
             // here we specify which method is used to render the label
+            // (associated_tostring deprecated, use associated_property option)
```
Comment (greg0ire) [inline Resources/doc/reference/action_list.rst]: "is deprecated"

**Comment by tim96** [comment]
@greg0ire sorry, I didn't see last commits. It is fixed in repository, but the site page https://sonata-project.org/bundles/admin/master/doc/reference/action_list.html not updated((( 
I close this PR. Thank you.

**Comment by greg0ire** [comment]
I knew this looked strangely familiar ;) . Thanks for following up.

**Pull request closed without merging**